osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -282,18 +283,12 @@ private void load(AudioManager audio, DialogOverlay dialog, OsuColour colours, S
             {
                 Schedule(() =>
                 {
-                    // if we have no beatmaps but osu-stable is found, let's prompt the user to import.
-                    if (!beatmaps.GetAllUsableBeatmapSetsEnumerable(IncludedDetails.Minimal).Any() && beatmaps.StableInstallationAvailable)
+                    // if we have no beatmaps, let's prompt the user to import from over a stable install if he has one.
+                    if (!beatmaps.GetAllUsableBeatmapSetsEnumerable(IncludedDetails.Minimal).Any() && stableImportManager.SupportsImportFromStable && DisplayStableImportPrompt)
```

Comment:
This line is a bit of a mouthful. I'd probably leave hanging indents after the `&&` operators at least, or maybe split to a method...

Reply:
Went with moving the `SupportsImportFromStable` check into `DisplayStableImportPrompt`; this should do aswell.